requirements.sh

```diff
@@ -8,5 +8,6 @@ else
 	if [ $? -eq 0 ]; then
 		sudo yum -y install epel-release
 		sudo yum install -y libmodbus
+		sudo yum install -y libmodbus-devel
```

Comment:
We should remove `sudo` before apt and yum commands as requirements script is called with sudo by the caller